types/datum_test.go

```diff
@@ -170,6 +170,12 @@ func (ts *testTypeConvertSuite) TestToInt64(c *C) {
 	binLit, err := ParseHexStr("0x9999999999999999999999999999999999999999999")
 	c.Assert(err, IsNil)
 	testDatumToInt64(c, binLit, -1)
+
+	ft = NewFieldType(mysql.TypeTiny)
```

Comment:
better to check other types(e.g., TypeShort, TypeLong, etc.)

Reply:
ok